src/Tizen.NUI.Components/Controls/Control.cs

```diff
@@ -320,6 +303,8 @@ private void Initialize()
             StateFocusableOnTouchMode = false;
 
             EnableControlState = true;
+
+            Feedback = true;
```

Comment:
After set it true as default, the control won't work in Linux backend.
We need to implement dummy feedback library for Linux or set it true only for fhub.

Reply:
If you want to enables it only in fhub, I think this will work.
```
#if PROFILE_MOBILE
Feedback = true;
#endif

```